src/.vuepress/public/openapi.css

```diff
@@ -0,0 +1,573 @@
+@charset "UTF-8";
+
+@font-face {
+  font-family: 'DM Sans';
+  src: url('./DMSans-VariableFont_opsz,wght.ttf') format('truetype');
```

Comment:
Could we just use some native font-family and avoid the 233kb extra in resources?

Reply:
It's part of the Bitwarden branding.